auth.mkd

```diff
@@ -807,9 +807,9 @@ The DRIP Wrapper has a specific use case for DRIP aware receivers. For receiver
 
 ### UAS RID Trust Assessment {#trust-assessment}
 
-As described in {{rid-trust}}, the receiver MUST perform verification of the data being received in Broadcast RID. This is because trust in a key is different from trust that an observed UA possesses that key. A chain of DRIP Links provides trust in a key. A message containing rapidly changing, unpredictable but sanity-checkable data, signed by that key, provides trust that some agent with access to that data also possesses that key. If the sanity check involves correlating physical world observations of the UA with claims in that data, then the probability is high that the observed UA is (or is collaborating with or observed in real time by) the agent with the key.
+As described in {{rid-trust}}, the receiver MUST perform validation of the data being received in Broadcast RID. This is because trust in a key is different from trust that an observed UA possesses that key. A chain of DRIP Links provides trust in a key. A message containing rapidly changing, unpredictable but sanity-checkable data, signed by that key, provides trust that some agent with access to that data also possesses that key. If the sanity check involves correlating physical world observations of the UA with claims in that data, then the probability is high that the observed UA is (or is collaborating with or observed in real time by) the agent with the key.
```

Comment:
```suggestion
As described in {{rid-trust}}, the receiver MUST perform validation of the data being received in Broadcast RID. This is because trust in a key is different from trust that an observed UA possesses that key. A chain of DRIP Links provides trust in a key. A sequence of messages containing changing, but sanity-checkable data, signed by the same key, provides trust that some agent with access to that data also possesses that key. If the sanity check involves correlating physical world observations of the UA with claims in that data, then the probability is high that the observed UA is (or is collaborating with or observed in real time by) the agent with the key.
```

This is the proposed changed by MCR for #44.

Reply:
I like MCR wording

Reply:
I stand by the argument that I have posted several times on the mailing list: "unpredictable" is a desirable quality of a cryptographic nonce; this validation that the observed UA actually possesses the key in question requires something conceptually similar to a cryptographic nonce. If over-ruled on this, I will shut up.